src/Simulation/EntryPointDriver/Mock/ErrorQuantumMachine.cs

```diff
@@ -15,14 +15,12 @@ namespace Microsoft.Quantum.EntryPointDriver.Mock
     /// </summary>
     internal class ErrorQuantumMachine : IQuantumMachine
     {
-        /// <summary>
+        // <summary>
         /// The target for the error quantum machine.
         /// </summary>
```

Comment:
What is the reason for this change? Accidentally deleted character? 
The fragments 
`// <summary>`
and 
`/// </summary>`
do not match. 
The former is the ordinary comment (`//`), the latter is the doc comment (`///`).

Reply:
This was an accidentally deleted character. Good catch. Fixed it in the latest push.